### ISSUE TITLE
Add skip/neutral option

### ISSUE THREAD

**Issue by mfoos** [jhudsl/papr#1]
As a bioinformatician whose best friend is a neuroscientist, standards very across fields and I am imagining bird ecologists getting bad swipes from statistical geneticists who just have a different frame of reference and/or lack of familiarity with the field. I know this is just for fun, but maybe you could pioneer some sort of 5th cardinal swipe!

**Comment by nstrayer** [comment]
I fully agree. This is on our radar and is going to be tackled very soon. Glad to get external validation for it.
